handlers/abstract_base_handler.py

```diff
@@ -1,14 +1,15 @@
 """The abstract base Handler class which all Handlers must subclass."""
 
 from abc import ABC, abstractmethod
+from comment import Comment
 
 
 class AbstractBaseHandler(ABC):
     """All Handlers must inherit from this AbstractBaseHandler class."""
 
     @classmethod
     @abstractmethod
-    def handle(cls, url):
+    def handle(cls, url) -> Comment:
```

Comment:
Same thing here. It's nice to have typing but I haven't played with type checking in Python before. Would be great to add this as a step in CI.

Reply:
The changes can be made progressively! Might be good to open another issue for it.